spatialpandas/geometry/multipolygon.py

```diff
@@ -158,6 +159,36 @@ def from_geopandas(cls, ga, orient=True):
         else:
             return mpa
 
+    @classmethod
+    def from_exterior_coords(cls, exterior_coords):
+        """
+        Build a spatialpandas MultiPolygonArray from exterior coordinates represented as a Python List or Numpy Array.
+
+        Args:
+            exterior_coords: A Python List where each entry contains the exterior coordinates of one of more Polygons.
+                             Each of these entries may be either a Python List or Numpy Array containing the exterior
+                             coordinates for a series of Polygons. Each Polygon in a List may have an arbitrary number
+                             of vertices but require a fixed number for a Numpy array.
+
+        Returns:
+            MultiPolygonArray
+
+        Note:
+            When using a Numpy Array is passed in, it is assumed that each entry is unfired, meaning that
+            all resulting MultiPolygons will have the same number of Polygons.
+
+        """
+        if isinstance(exterior_coords, (list, np.ndarray)):
+            if isinstance(exterior_coords, np.ndarray):
+                warnings.warn(f"Constructing a MultiPolygonArray using a Numpy Array with a fixed shape {exterior_coords.shape}."
+                              f"Each MultiPolygon will contain exactly {exterior_coords.shape[0]} Polygons")
+            mpa = [[[arr.ravel()] for arr in exterior] for exterior in exterior_coords]
```

Comment:
If a wrong input shape is passed as a `exterior_coords`, this could give a non-helpful error message.